releases/upcoming.rst

```diff
@@ -0,0 +1,14 @@
+.. |br| raw:: html
+
+   <br />
+
+Upcoming
+========
+
+These are changes yet unreleased. If you'd like to try them out before a new official release, you can use a `nightly release <https://github.com/PostgREST/postgrest/releases/tag/nightly>`_.
```

Comment:
Hm, nightlies are different now. Let's remove this for now.
```suggestion
```

Reply:
@monacoremo So as we discussed in https://github.com/PostgREST/postgrest/pull/1901#issuecomment-892939818, for trying the change here we'd need to link to [postgrest/actions/runs/1128881855](https://github.com/PostgREST/postgrest/actions/runs/1128881855) and tell users to download the artifacts right?

Some questions here:

- Do you know the artifacts' duration? I assume Github assigns them a lifetime.
- Try visiting the [run page](https://github.com/PostgREST/postgrest/actions/runs/1128881855) as incognito(not logged to GitHub), can you download the artifacts? I can't :confused:, also I wonder if logged-in non-team members can download them. Do you know if there's another way to link to the artifacts?

Reply:
You need a Github account with read permissions on the repository (so any account as we are public, and anyone on Gitter), the artifacts are retained for 90 days: https://docs.github.com/en/actions/managing-workflow-runs/downloading-workflow-artifacts

For 'power users' to try a specific change right away (e.g. for whoever submitted the issue that got fixed), we should instruct them to go via the check on the relevant commit.

In order to have something to link to long term, we should tag a new version. In this case this might warrant a proper 8.0.1 release even. Otherwise I suggest 9.0.0-a1, which will create a prerelease, to be followed up with -a2 etc. The commit and date will be available on the release page. Or without the 1, just 9.0.0-a, if we want to overwrite the pre-release with new tags as development progresses. 

So for the instructions here I suggest:

 * To test unreleased changes, use one of the latest pre-releases on the release page (link)
 * If you need absolute bleeding edge (e.g. because you want to test if a specific commit fixed your issue), use the artifacts from the respective or latest CI run: https://github.com/PostgREST/postgrest/actions/workflows/ci.yaml?query=branch%3Amain (filtered for main branch)